e2e/framework/util.go

```diff
@@ -42,6 +42,38 @@ func (f *Framework) WaitForPodRunning(podName string, ns string) error {
 	})
 }
 
+func (f *Framework) WaitForPersistentVolumeClaimBound(pvcName, ns string) error {
+	return wait.PollImmediate(time.Second, PollTimeout, func() (bool, error) {
+		pvc, err := f.HostClient.CoreV1().PersistentVolumeClaims(f.VclusterNamespace).Get(f.Context, pvcName+"-x-"+ns+"-x-"+f.Suffix, metav1.GetOptions{})
```

Comment:
Instead of `pvcName+"-x-"+ns+"-x-"+f.Suffix`, please use the function`translate.PhysicalName()`